apps/sms/services/js/drafts.js

```diff
@@ -138,6 +149,13 @@
     },
 
     /**
+     * Returns list of all thread less drafts.
+     */
+    getAllThreadless: function () {
+      return draftIndex.get(null) || [];
```

Comment:
> mmm it shouldn't, Drafts.request is doing draftIndex = new Map(records || []); ?

Yes! However we have  ```Drafts.byDraftId()``` that is internally does  ```draftIndex.get(null)``` and returns ```undefined```. Nevermind we use this method only if we have ```draftId```, and in inbox we iterate through _entire map_ - so it's good, we don't have issue in master :)

> Where was it failing ? in B2G Desktop ?

Now everywhere, but only with my patch because I request _only_ thread less drafts that involves ```draftIndex.get(null)```, so we still need this ```|| []``` for new method :)